src/nvim/api/keysets.lua

```diff
@@ -50,6 +50,7 @@ return {
     "callback";
     "desc";
     "replace_keycodes";
+    "is_abbrev";
```

Comment:
"is_" prefix is not necessary
```suggestion
    "abbrev";
```

Reply:
I just used it to keep compatibility with the internal naming, if that's not a problem, I can commit the suggestion.

Reply:
Nope, how it's internally called is not a problem. Just `abbrev` would correspond to the `:abbrev` command.